week-1/errors/3.js

```diff
@@ -1,7 +1,9 @@
-const cardNumber = 4533787178994213;
-const last4Digits = cardNumber.slice(-4);
+const cardNumber = "4533787178994213";
+const last4Digits = cardNumber.slice(12,16);
```

Comment:
You can keep `.slice(-4)`, that will still work. The only issue here was the string 🙂